warehouse/pom.xml

```diff
@@ -445,6 +445,11 @@
             <groupId>gov.nsa.datawave</groupId>
             <artifactId>datawave-common-test</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
You don't need this.

Reply:
If you look at the recent commit history you can see how this was removed, and then added back in when other modules failed to build.

Reply:
i was wrong.  if we keep this, all child modules will inherig junit-jupiter-engine, which is what we want.  sorry for the confusion

Reply:
all good!